infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -230,14 +243,19 @@ public void handleTokenQuery(CorfuPayloadMsg<TokenRequest> msg,
     /**
      * Service an incoming token request.
      */
-    @ServerHandler(type=CorfuMsgType.TOKEN_REQ)
+    @ServerHandler(type=CorfuMsgType.TOKEN_REQ, opTimer=metricsPrefix + "token-req")
     public synchronized void tokenRequest(CorfuPayloadMsg<TokenRequest> msg,
-                                          ChannelHandlerContext ctx, IServerRouter r) {
+                                          ChannelHandlerContext ctx, IServerRouter r,
+                                          boolean isMetricsEnabled) {
         TokenRequest req = msg.getPayload();
 
+        boolean isEnabled = MetricsUtils.isMetricsCollectionEnabled();
```

Comment:
this is the only place I saw where the conditional should have been used, and isn't